01_todo_app/web/src/components/landing/index.jsx

```diff
@@ -1,68 +1,133 @@
+                            </h1>
+                        </div>
+                        <div className="flex justify-center align-center my-10">
+                            <div className="md:w-1/2">
+                                <p className="block text-normal text-gray-300">
+                                    The ToDo list app is a demo showcasing the capability of running a simple app with a backend, frontend
+                                    and a database on the Golem Network. The purpose of this app is to demonstrate the possibility of
+                                    running these services on the network.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <form
+                            onSubmit={(e) => {
+                                addNewTodo(e)
+                                setTask("")
+                            }}
+                        >
+                            <div>
+                                <p className="text-xs text-yellow-600 mt-1 rounded-md">
+                                    Please be aware that data inside this app is not persisted, and will be lost at the end of the demo!
+                                </p>
+                                <div className="mt-1 flex rounded-md shadow-sm">
+                                    <input
+                                        type="text"
+                                        name="text"
+                                        required
+                                        value={task}
+                                        onChange={(event) => setTask(event.target.value)}
+                                        placeholder="Type your task"
+                                        className="px-6 text-lg py-4 block w-full rounded-none rounded-l-md bg-input text-white placeholder:text-gray-300"
+                                    />
+
+                                    <button
+                                        type="submit"
+                                        className="relative -ml-px inline-flex items-center space-x-2 rounded-r-md text-white text-lg bg-button px-8 md:px-24 py-2 "
+                                    >
+                                        Add
+                                    </button>
+                                </div>
+                            </div>
+
+                            <div>
+                                <input type="hidden" value="NEW" name="status" />
+                            </div>
+                        </form>
+                    </div>
+                    <div className="grid grid-cols-12 mt-20 gap-x-8 md:gap-x-0 gap-y-10">
+                        {messages.map((message) => (
+                            <>
```

Comment:
Here using `<>` is justified because you have many `div`. Although, instead of `returning a group of divs`, it would be more proper to return a `TaskListItem` component instance.